content/_includes/TrueCommandLDAP.md

```diff
@@ -0,0 +1,71 @@
+Users can log in with an LDAP or AD account without creating a separate TrueCommand login.
+
+LDAP and AD require the server IP address or DNS hostname and domain to use.
+The LDAP or AD Username (optional) is required when the TrueCommand user name does not match the LDAP or AD credentials.
+
+Click on the **<i class="material-icons" aria-hidden="true" title="Settings">settings</i>&nbsp; (Gear) > Administration**.  
+Click **Add**  on the **LDAP Servers** widget to open the **Add LDAP Server** configuration screen.
+
+![AdministrationLDAPServersWidget](/images/TrueCommand/2.3.2/AdministrationLDAPServersWidget.png "Administration Screen LDAP Servers Widget")
+
+To configure LDAP, type the LDAP server IP address or DNS host name into the **LDAP Server URL** field, type the domain name in the **Domain** field, and click **ADD SERVER**.
+You can add multiple LDAP servers and domains. 
+
+The **Test LDAP Config** <span class="iconify" data-icon="mdi:test-tube"></span> icon opens a window that allows you to test your connection to the LDAP server. The **Remove LDAP Server** <span class="iconify" data-icon="mdi:delete"></span> icon removes the selected LDAP server. 
+
+![AddLDAPServerScreen](/images/TrueCommand/2.3.2/AddLDAPServerScreen.png "Add LDAP Server")
+
+| Setting | Description |
+|---------|-------------|
+| **Hostname** | (Required) Enter the host name, IP or DNS name, of the LDAP server, with port number on the end. For example: *ldap.mycorp.com:636* (SSL port is typically 636 for AD/LDAP). |
+| **Domain** | (Required) Base domain settings of the user. For example: <i>dc=mycorp,dc=com</i> for a typical `username@mycorp.com` user account. |
+| **Group Domain** | Enter the alternative domain setting to use when searching for groups. The default value is the same as **Domain**. |
+| **Verify SSL** | Select to require strict SSL certificate verification. The default value is false. Disable this option if the host name of the system is different than the one listed on the SSL certificate, an IP is used for the connection instead of the DNS host name, or if a self-signed certificate is used by the LDAP server. |
+| **User ID Field** | Enter the user ID for the user that logs in (this is class-matched to the login username). Enter **Domain** name to use for user-matching. The default value is **uid** (user ID). Another field commonly-used is **cn** (common name). |
+| **Group ID Field** | Enter the class for finding groups associated with user. Default is cn (common name). Enter the **Domain** name to use when searching for a group name. |
+| **BIND User Domain** | Enter the full domain setting for a pre-authenticated bind to the server. For example: <i>uid=binduser,cn=read-only-bind,dc=mycorp,dc=com</i>. For an unauthenticated bind enter just a name (example: <i>truecommand-bin</i>). This is sometimes used for logging purposes on the LDAP, but otherwise is not validated. |
```

Comment:
```suggestion
| **BIND User Domain** | Enter the full domain setting for a pre-authenticated bind to the server. For example: <i>uid=binduser,cn=read-only-bind,dc=mycorp,dc=com</i>. For an unauthenticated bind, enter just a name (example: <i>truecommand-bin</i>). This is sometimes used for logging purposes on the LDAP but otherwise is not validated. |
```